commonTestResources/exampleOpenApiFiles/valid/basePathVariations/basePathDefined.yml

```diff
@@ -0,0 +1,14 @@
+swagger: '2.0'
```

Comment:
to be more consistent with OpenAPI 3 `serversDefinedDifferently`, this folder could be `basePathDefinedDifferently`

Reply:
Included in [Changed 'urlErrorCodes' to 'validationErrorCodes' ](https://github.com/RuntimeTools/OpenAPIValidators/pull/172/commits/35079f970d8c26ec41077ad2d3850bdddad48f5a)